gulpfile.js

```diff
@@ -1,170 +1,226 @@
-	.pipe(sass())
-	.pipe(cssnano({ autoprefixer: false, zindex: false }))
-	.pipe(rename({ suffix: '.min' }))
-	.pipe(gulp.dest(`${DIST_FOLDER}/css`)));
-
-gulp.task('minify-js', ['process-master-js', 'process-homepage-js', 'process-app-js', 'move-page-specific-js'], () => gulp.src([`${DIST_FOLDER}/js/**/*.js`])
-	.pipe(uglify())
-	.on('error', (err) => { util.log(util.colors.red('[Error]'), err.toString()); })
-	.pipe(rename({ suffix: '.min' }))
-	.pipe(gulp.dest(`${DIST_FOLDER}/js`)));
-
-
-gulp.task('process-app-js', () => {
-	// const appRootFolder = 'js/apps';
-	// const appFolders = fs.readdirSync(appRootFolder).filter((file) => {
-	// 	return fs.statSync(path.join(appRootFolder, file)).isDirectory();
-	// });
-
-	// appFolders.forEach((folder) => {
-	// 	gulp.src([
-	// 		`js/apps/${folder}/app.js`,
-	// 		`js/apps/${folder}/constants.js`,
-	// 		`js/apps/${folder}/config.js`,
-	// 		`js/apps/${folder}/filters/**/*.js`,
-	// 		`js/apps/${folder}/dataServices/**/*.js`,
-	// 		`js/apps/${folder}/services/**/*.js`,
-	// 		`js/apps/${folder}/controllers/**/*.js`,
-	// 		`js/apps/${folder}/directives/**/*.js`
-	// 	])
-	// 		.pipe(jshint({
-	// 			esversion: 6
-	// 		}))
-	// 		.pipe(jshint.reporter(stylish))
-	// 		.pipe(babel({
-	// 			presets: ['es2015']
-	// 		}))
-	// 		.pipe(stripCode({
-	// 			start_comment: 'test-code',
-	// 			end_comment: 'end-test-code'
-	// 		}))
-	// 		.pipe(concat('app.js'))
-	// 		.pipe(gulp.dest(`${DIST_FOLDER}/js/apps/${folder}`));
-	// });
+const babel = require("gulp-babel");
+const clean = require("gulp-clean");
+const concat = require("gulp-concat");
+const coveralls = require("gulp-coveralls");
+const cssnano = require("gulp-cssnano");
+const fs = require("fs");
+const gulp = require("gulp");
+const jshint = require("gulp-jshint");
+const path = require("path");
+const rename = require("gulp-rename");
+const runSequence = require("run-sequence");
+const sass = require("gulp-sass");
+const stripCode = require("gulp-strip-code");
+const stylish = require("jshint-stylish");
+const uglify = require("gulp-uglify");
+const util = require("gulp-util");
+
+const DIST_FOLDER = "public/dist";
+
+gulp.task("clean", () => gulp.src(`${DIST_FOLDER}`).pipe(clean()));
+
+gulp.task("process-scss", () =>
+  gulp
+    .src(["stylesheets/restaurant-week.scss"])
+    .pipe(sass())
+    .pipe(cssnano({ autoprefixer: false, zindex: false }))
+    .pipe(rename({ suffix: ".min" }))
+    .pipe(gulp.dest(`${DIST_FOLDER}/css`))
+);
+
+gulp.task(
+  "minify-js",
+  [
+    "process-master-js",
+    "process-homepage-js",
+    "process-app-js",
+    "move-page-specific-js"
+  ],
+  () =>
+    gulp
+      .src([`${DIST_FOLDER}/js/**/*.js`])
+      .pipe(uglify())
+      .on("error", err => {
+        util.log(util.colors.red("[Error]"), err.toString());
+      })
+      .pipe(rename({ suffix: ".min" }))
+      .pipe(gulp.dest(`${DIST_FOLDER}/js`))
+);
+
+gulp.task("process-app-js", () => {
+  // const appRootFolder = 'js/apps';
```

Comment:
Is this all commented out, but left in on purpose?

Reply:
yes, i plan on using all of your tasks...but i'm not ready to add any angular in yet.